yarn-project/boxes/token/src/tests/token.contract.test.ts

```diff
@@ -489,6 +489,28 @@ describe('e2e_token_contract', () => {
           );
         });
 
+        it('transfer on behalf of other after deleting approval', async () => {
```

Comment:
Not sure this test is required. 

It is fairly obvious that this will be a noop, as i am adding a witness from two pxes then removing it from one, then making a call with the second pxe that succeeds. 

The second test is the only one that actually tests what this PR does.